tests/integration/testlib/assertlib.py

```diff
@@ -53,3 +53,8 @@ def assert_absent(*ifnames):
 
     current_state = statelib.show_only(ifnames)
     assert not current_state[INTERFACES]
+
+
+def assert_all_equal(iface_macs):
+    """ Assert that all the interfaces have same mac address """
```

Comment:
This function is not specific to macs, so it should not mention it in the docstring.